README.md

```diff
@@ -1,31 +1,32 @@
-# django-mate
-# ✨ hacktoberfest2022 💫
+# dev-mates
+# ✨ hacktoberfest10-2023 💫
 
-![banner](https://hacktoberfest.com/_next/static/media/opengraph.da6e44c0.png)
+![banner](https://github.com/FerdausPolok/dev-mates/assets/68804751/850772fe-43f2-4472-91c2-96933ce93d7f)
```

Comment:
I guess we can use their official banner here?

Reply:
Sure. Updated, thanks.